book_test.js

```diff
@@ -1,52 +0,0 @@
-const books = JSON.parse(localStorage.getItem('books')) || [];
```

Comment:
Great job team 🤝

- [ ] [Optional] I would recommend deleting this file since it is not being used in this project.